DND_Manager/settings.py

```diff
@@ -34,6 +34,7 @@
     'Campaign_Manager.apps.CampaignManagerConfig',
     'Character_Builder.apps.CharacterBuilderConfig',
     'Users.apps.UsersConfig',
+    'Campaign_Manager.apps.CampaignManagerConfig',
```

Comment:
I tried to test the application and this line caused an error because it duplicates line 34. Please delete